src/components/shared/page.tsx

```diff
@@ -129,6 +135,25 @@ const getImageMappings = (imageSalt: string, capi: Content): ImageMappings => {
         return { ...blockMappings, ...getContributorMappings(capi.tags, imageSalt) }
 }
 
+const liveblogHydrationProps = (capi: Content): Content => {
+    const { id: contentId, type, webTitle, webUrl, apiUrl, fields, tags, references, isHosted, blocks } = capi;
+    const { main, body, totalBodyBlocks } = blocks as Blocks;
+    const { createdBy, lastModifiedBy, ...mainBlocks } = main as Block;
```

Comment:
Are `createdBy` and `lastModifiedBy` used here?

Reply:
this was a way to pull them out and use all the other fields, blocks have a lot of required fields so this seemed to make sense